.gitignore

```
@@ -3,12 +3,12 @@
 # See https://help.github.com/articles/ignoring-files/ for how to configure your environment.
 # See https://github.com/github/gitignore/tree/master/Global for a list of global ignore rules.
 
-/node_modules/
-/dist/
+**/node_modules/
```

The `node_modules` is supposed to be at top-level.
```suggestion
node_modules/
```